### DISCUSSION TITLE
Belt idlers, where to find them?

### DISCUSSION THREAD

**Discussion by mrThe** [bassamanator/everything-sovol-sv06#52] [Q&A]
Thats the question. Where can i find the replacement idler? I fit one on the X asis, but i can't find anything for Y, as fit is way to tight so it may now sit on place, or it will provide zero tension on the belt.

![image](https://github.com/user-attachments/assets/df63b19a-3e2b-46e4-a795-58481ccbe41b)

**Comment by bassamanator** [comment]
Should be easy to find on aliexpress, even amazon.

**Reply by mrThe** [reply]
Sadly it's not. I've ordered a few from ali - none of them fit as it should. I've even asked sellers for assistance on picking the right one - every seller said that there is no such idler for this drawing.

**Reply by bassamanator** [reply]
The belts are 6mm\*\*  in width, and the bore hole is 4mm. Those are the only figures that matter. 

\*\* Yes, Sovol choose the wrong size for the idles, 8mm.

These *should* be fine:
- https://www.aliexpress.com/item/1005005484139954.html
- https://www.aliexpress.com/item/1005005921593580.html

**Reply by mrThe** [reply]
<img width="247" alt="image" src="https://github.com/user-attachments/assets/59a174f4-aad0-4c25-8e4a-7bba40e25790">

There is a thing about this 13.6mm part. It should be at least 13mm, if it gets any less - it will not work for Y axis, since it will provide no tension. And unfortunately there is a now way to check this size on items you linked.

I've already orders a few items and none of them fits. More to it - i've ordered even more and now waiting for shipment to arrive, will get back to this topic in case it fits.

**Reply by bassamanator** [reply]
> There is a thing about this 13.6mm part. It should be at least 13mm, if it gets any less - it will not work for Y axis, since it will provide no tension. And unfortunately there is a now way to check this size on items you linked.

Belt tension is adjustable...

**Reply by mrThe** [reply]
It is, but it have it's limit. And there is two options around it: change(shorten) the belt or change the idler holder. I don't like both. 

Tho now i'm thinking that shortening the belt should be an easy task, as Y belt(as well as X belt) is not looped and just holded by two pieces under the bed..

Well i guess this is the solution, will definetly check it later and come back with the result

**Reply by bassamanator** [reply]
Again, belt tension is adjustable via the knob at the front and by how much slack you leave in the belt when 'fix' the belt into place under the bed. You appear to have a lot of slack in the belt when the knob at the front is fully loose.

It might help to use a part such as [this](https://www.printables.com/model/884789-wartungsstander-sovol-sv06-maintenance-stand) when you're doing adjustments. Also see the belt tensions points in the initial steps on this repo.

**Reply by mrThe** [reply]
Thanks for the link to the part, looks pretty convenient. 

Knob was the initial issue for me - if idler have wrong size knob just hits its limit and belt still have no tension at all. But as i mentioned above - i can just shorten the belt.

I guess sometimes i need to ask stupid questions to understand that i just ignored the most obvious solution to the problem.

**Comment by Jonybat** [comment]
If you want to try to source the bearings and print the shell instead: https://www.printables.com/model/842701-gt2-toothed-idler-for-sv06

**Reply by mrThe** [reply]
Saw them, but i doubt it's a good solution considering it's always working under belt tension

**Reply by Jonybat** [reply]
Been using for months, still work fine. Not an engineer, but the amount of tension on those belts is nowhere near enough to cause any strain on the plastic. If the tension is indeed that high, you will probably damage the stepper shaft or some other part of the printer before you break this little plastic wheel.

**Reply by mrThe** [reply]
Thanks, i guess i'm gonna give it a try, if i don't find anything.
